key.core/src/main/java/de/uka/ilkd/key/speclang/jml/translation/JMLSpecFactory.java

```diff
@@ -628,8 +628,8 @@ private Term translateUnionClauses(Context context, ImmutableList<ProgramVariabl
             if (translated.equalsModIrrelevantTermLabels(tb.strictlyNothing())) {
                 if (originalClauses.size() > 1) {
                     throw new SLTranslationException(
-                        "\"assignable \\less_than_nothing\" does not go with other "
-                            + "assignable clauses (even if they declare the same).",
+                        "\"modifiable \\less_than_nothing\" cannot be joined with other "
```

Comment:
I think I have never seen this message so far. However, I think it would be a good idea to explicitly say `\\strictly_nothing` here (or are there multiple different `\\less_than_nothing` terms?).

Reply:
`\less_than_othing` is the old name of `\strictly_nothing`

Reply:
Thanks for the feedback, I fixed it (and also renamed one obsolete mention of the old version).